less/variables.less

```diff
@@ -11,4 +10,11 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+// border
+@border: 2px dashed @navigation-border;
+
+// breakpoint
+@mobileBreakpoint: 500px;
+@tabletBreakpoint: 800px;
```

Comment:
You missed a step here. instead of just doing `@mobileBreakpoint: 500px;` you could want to do something like `@mobile: ~"(max-width: 500px)"; `. Then whenever you want to set up a media query is just as easy as doing 
`@media @mobile {...}`